assets/src/blocks/Columns/ColumnsEditor.js

```diff
@@ -0,0 +1,185 @@
+import {
+  CheckboxControl,
+  PanelBody,
+  RangeControl
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import { InspectorControls } from '@wordpress/block-editor';
+
+import { URLInput } from '../../components/URLInput/URLInput';
+import { EditableColumns } from './EditableColumns';
+import { Columns } from './Columns';
+import { MAX_COLUMNS_AMOUNT, MIN_COLUMNS_AMOUNT } from './ColumnConstants';
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+
+const renderEdit = (attributes, toAttribute, setAttributes, isSelected) => {
+  const { columns } = attributes;
+
+  const addColumn = () =>
+    setAttributes({
+      columns: [
+        ...columns,
+        {
+          title: '',
+          description: '',
+          attachment: 0,
+          cta_link: '',
+          cta_text: '',
+          link_new_tab: '',
+        }
+      ]
+    });
+
+  const removeColumn = () => setAttributes({ columns: columns.slice(0, -1) });
+
+  if (!isSelected) {
+    return null;
+  }
+
+  return (
+    <>
+      <InspectorControls>
+        <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
```

Comment:
`Settings` ?

Reply:
It's been bugging me as well, but we write `Setting` for other blocks so I kept it 😅